csv_permissions/permissions.py

```diff
@@ -354,15 +321,28 @@ def is_global_perm(self, perm: str) -> bool:
 
     def has_perm(self, user: Model, perm: str, obj: Model) -> bool:
         try:
-            user_group = str(user.get_profile().user_type)
+            # TODO: don't hardcode this
+            user_type = str(user.get_profile().user_type)
```

Comment:
apart from this it seems like csv_permissions is compatible with any user implementation?

what about just a `resolve_user_type` setting?

Reply:
^ ya, means if it's vanilla group we can still do

Reply:
There is now a `CSV_PERMISSIONS_GET_USER_TYPE` setting that defaults to returning a function that resolves to `return user.user_type`, so this has been removed